proposals/forms.py

```diff
@@ -816,11 +836,9 @@ def __init__(self, *args, **kwargs):
 
         super(ProposalSubmitForm, self).__init__(*args, **kwargs)
 
-        self.fields["inform_local_staff"].label_suffix = ""
-
-        self.fields["inform_local_staff"].label = mark_safe(
-            self.fields["inform_local_staff"].label
-        )
+        self.fields["inform_local_staff"].label = self.fields[
+            "inform_local_staff"
+        ].label
```

Comment:
For me, this part is not necessary to fix the bug.

But it's also harmless.

Reply:
The same goes for the `other_stakeholders` part, which never gave us any trouble. I have no rational reasoning but feel like keeping it in just to be safe.